student.rb

```diff
@@ -1,15 +1,11 @@
-class Student
-  attr_reader :age, :phase
-  attr_accessor :name
+require_relative 'human'
+
+class Student < Human
+  attr_reader :phase
```

Comment:
This can go up to the superclass too.  The reader method definition doesn't need to be coupled with the place where the instance variable is initialized.